pomegranate(new).prefab

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We don't want different versions for same files. Learn to use git so that you can update changes in the previous file and commit them to your existing branch.
Remove this file and shift those changes to the old file